src/DocLayer.actor.cpp

```diff
@@ -877,7 +913,27 @@ int main(int argc, char** argv) {
 	init_document_error();
 
 	g_network = newNet2(NetworkAddress(), false);
+#ifndef TLS_DISABLED
+	try {
+		if (tlsCertPath.size())
+			tlsOptions->set_cert_file(tlsCertPath);
+		if (tlsCAPath.size())
+			tlsOptions->set_ca_file(tlsCAPath);
+		if (tlsKeyPath.size()) {
+			if (tlsPassword.size())
+				tlsOptions->set_key_password(tlsPassword);
+
+			tlsOptions->set_key_file(tlsKeyPath);
+		}
+		if (tlsVerifyPeers.size())
+			tlsOptions->set_verify_peers(tlsVerifyPeers);
 
+		tlsOptions->register_network();
+	} catch (Error& e) {
+		printf("Error: %s \n", e.what());
```

Comment:
Should this be `fprintf(stderr, )` instead?

Reply:
Sure thing.